netbench/cdk/lambda/exportS3.js

```diff
@@ -0,0 +1,21 @@
+const AWS = require('aws-sdk')
+const cloudconfig = {
+  apiVersion: '2014-03-28',
+  region: process.env.REGION, 
+}
+const cloudwatchlogs = new AWS.CloudWatchLogs(cloudconfig)
+exports.handler =  async (event, context) => {
+   const params = {
+    destination: process.env.BUCKET_NAME, 
+    from: new Date().getTime() - 600000, //ten minutes, will add exact time stamp in future iteration
+    logGroupName: process.env.LOG_GROUP_NAME,
+    to: new Date().getTime()
+  };
+
+var data = await cloudwatchlogs.createExportTask(params).promise();
+
+return ({
+    body: data
+});
```

Comment:
* spacing is a bit weird
* use `const` when variables aren't reassigned
* you can use short-hand field initializers

```suggestion
  const body = await cloudwatchlogs.createExportTask(params).promise();

  return { body };
```